### PR TITLE
Jon/fix/android-fingerprint-switch

### PR DESCRIPTION
**Pull request by Jon-edge**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Description

 none

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208333397037146

### CODE REVIEW

File: src/components/scenes/PinLoginScene.tsx [hunk 1]
```diff
@@ -91,17 +101,33 @@ export function PinLoginScene(props: Props) {
...
-      userInfo?.username ??
-      sprintf(lstrings.guest_account_id_1s, loginId.slice(loginId.length - 3)),
-    [loginId, userInfo]
-  )
-
   // ---------------------------------------------------------------------
   // Effects
   // ---------------------------------------------------------------------
 
+  const handleTouchLogin = async (userInfo: EdgeUserInfo): Promise<void> => {
```
Comment (swansontec) [inline src/components/scenes/PinLoginScene.tsx:108]: Why was this moved to the "Effects" section? It's a handler, so it should stay in the "Handlers" section. I don't see any dependencies that have changed, so the location shouldn't matter.

**Review by swansontec — APPROVED** [review]
Please move the handler back to the right place, but the code changes make sense.

File: src/components/scenes/PinLoginScene.tsx (hunk 1 above)
Comment (Jon-edge) [reply src/components/scenes/PinLoginScene.tsx:108]: I was getting a error about using before declaration
